### PR TITLE
1605.release rc2

### PR DESCRIPTION
**Pull request by daira**
Release 1.0.0-rc2. ref #1605

### CODE REVIEW

File: doc/release-process.md [hunk 1]
```diff
@@ -106,15 +108,13 @@ Notify the Zcash DevOps engineer/sysadmin that the release has been tagged. They
 
 Then, verify that nodes can connect to the testnet server, and update the guide on the wiki to ensure the correct hostname is listed in the recommended zcash.conf.
 
-## F. publish the release announcement (blog, zcash-dev, slack)
-## G. celebrate
+## F. Update the Beta Guide
```
Comment (ageis) [inline doc/release-process.md]: Can we add post-release steps of running the Gitian build and building the .deb packages?

**Comment by ageis** [comment]
Aside from my single comment, looks great to me. Excellent work @daira!

**Comment by bitcartel** [comment]
Is testnet being reset?  Re: pchmessagestart string update

**Comment by daira** [comment]
@ageis ACK'd e2a227a.

@zkbot r+

**Comment by daira** [comment]
Yes, testnet is being reset.

**Pull request merged**